semantic-source/src/Source/Language.hs

```diff
@@ -0,0 +1,136 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE OverloadedStrings #-}
+module Source.Language
```

Comment:
This seems to be missing any corresponding removal of `Data.Language` from `semantic`; is the intention to migrate stuff separately after we release the new version of `semantic-source` to hackage?

Reply:
Correct.